ci-operator/step-registry/openstack/test/dpdk/openstack-test-dpdk-commands.sh

```diff
@@ -168,7 +171,7 @@ else
     exit 1
 fi
 
-TESTPMD_OUTPUT=$(oc -n "${CNF_NAMESPACE}" rsh "${CNF_POD}" bash -c "yes | testpmd -l 2-3 --in-memory --allow 00:04.0 --socket-mem 1024 -n 4 --proc-type auto --file-prefix pg  -- --disable-rss  --nb-cores=1 --rxq=1 --txq=1 --auto-start --forward-mode=mac")
+TESTPMD_OUTPUT=$(oc -n "${CNF_NAMESPACE}" rsh "${CNF_POD}" bash -c "yes | testpmd -l 2-3 --in-memory --allow ${DPDK_PCI_DEVICE} --socket-mem 1024 -n 4 --proc-type auto --file-prefix pg  -- --disable-rss  --nb-cores=1 --rxq=1 --txq=1 --auto-start --forward-mode=mac")
```

Comment:
this one I'm not sure if it'll be happy. @zivgreenberg if you can comment on this one. Thanks

Reply:
in other words, do I need to trim the PCI device to remove the first two 00 ?

Reply:
@EmilienM There is no need to trim. It should work fine as long as the bus address is correct.